styles/style-main.css

```diff
@@ -0,0 +1,95 @@
+.navbar {
+  display: flex;
+  justify-content: center;
+  background-color: beige;
+}
+
+.navbar ul {
+  display: flex;
+  align-self: center;
+  width: 40vw;
+  padding: 10px;
+  justify-content: center;
+}
+
+.navbar ul > * {
+  flex: 1 1 25%;
+  list-style: none;
+  text-align: center;
+  font-size: 30px;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+a {
+  text-decoration: none;
+}
+
+main {
+  display: block;
+}
+
+section {
+  display: flex;
+}
+
+.headline-left {
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 60%;
+}
+
+.headline-name {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: 600;
+  font-size: 7em;
+  margin-left: 150px;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
+.headline-bio {
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] While you got the correct font family here, the link on your `<head></head>` shows me that you got Poppins light, kindly change it to Poppins regular. Since the font is provided via web resource, changing the font-weight as you do in the line below without having the one with correct weight on the `<head></head>` doesn't achieve the same effect